TMDS_3/include/CGAL/tetrahedron_soup_to_triangulation_3.h

```diff
@@ -0,0 +1,170 @@
+// Copyright (c) 2009-2014 INRIA Sophia-Antipolis (France).
+// Copyright (c) 2010-2013 GeometryFactory Sarl (France).
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org).
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: GPL-3.0-or-later OR LicenseRef-Commercial
+//
+//
+// Author(s)     :  Mael Rouxel-Labbé, Maxime Gimeno, Jane Tournois
+//
+//******************************************************************************
+// File Description :
+//******************************************************************************
+
+#ifndef CGAL_TMDS_3_TETRAHEDRON_SOUP_TO_C3T3_H
+#define CGAL_TMDS_3_TETRAHEDRON_SOUP_TO_C3T3_H
+
+#include <CGAL/license/TMDS_3.h>
+
+#include <CGAL/TMDS_3/tet_soup_to_c3t3.h>
+#include <CGAL/TMDS_3/internal/TMDS_3_helper.h>
+#include <CGAL/boost/graph/named_params_helper.h>
+
+#include <vector>
+#include <array>
+#include <boost/unordered_map.hpp>
+
+namespace CGAL {
+
+  /** \ingroup PkgTMDS3Functions
+  * builds a 3D triangulation from a soup of tetrahedra.
+  *
+  * @tparam TetrahedronRange a model of `Range` whose value type is
+  * a `Tetrahedron_3`
+  * @tparam Triangulation a 3D triangulation class that has
+  * a vertex base model of `MeshVertexBase_3`
+  * and a cell base model of `MeshCellBase_3`
```

Comment:
The documentation should say that the point type of the tetrahedron must convertible to `Triangulation::Point`.